src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py

```diff
@@ -1569,7 +1585,6 @@ def quantize_torch_qat_export(
     _convert_quantizable_gemm_no_activations(model)
     _quantize_qat_embedding(model)
     quantize_resnet_identity_add_inputs(model)
-    quantized_residual_add_optim(model)
```

Comment:
let's delete this function if it is no longer used elsewhere in sparseml

Reply:
Done